contribute/backend/services.md

```diff
@@ -109,30 +109,30 @@ func InitializeForTest(cla setting.CommandLineArgs, opts Options, apiOpts api.Se
 
 A background service runs in the background of the lifecycle between Grafana startup and shutdown. To run your service in the background, it must satisfy the `registry.BackgroundService` interface. Pass it through to the `NewBackgroundServiceRegistry` call in the [ProvideBackgroundServiceRegistry](/pkg/registry/backgroundsvcs/background_services.go) function to register it.
 
-You can see an example implementation above of the Run method.
+For an example of the `Run` method, see the previous example.
 
 ## Disabled services
 
-If you want to guarantee that a background service is not run by Grafana when certain criteria are met/service is disabled, your service must satisfy the `registry.CanBeDisabled` interface. When the `service.IsDisabled` method returns true, Grafana will not call the `service.Run` method.
+If you want to guarantee that a background service is not run by Grafana when certain criteria are met, or if a service is disabled, your service must satisfy the `registry.CanBeDisabled` interface. When the `service.IsDisabled` method returns `true`, Grafana won't call the `service.Run` method.
 
 If you want to run certain initialization code whether service is disabled or not, you need to handle this in the service factory method.
 
-You can see an example implementation above of the `IsDisabled` method and custom initialization code when the service is disabled.
+For an example of the `IsDisabled` method and custom initialization code when the service is disabled, see the previous implementation code.
 
-## Run Wire / generate code
+## Run Wire (generate code)
 
-Running `make run` calls `make gen-go` on the first run. `gen-go` in turn calls the wire binary and generates the code in [wire_gen.go](/pkg/server/wire_gen.go) and [wire_gen.go](/pkg/cmd/grafana-cli/runner/wire_gen.go). The wire binary is installed using [bingo](https://github.com/bwplotka/bingo) which downloads and installs all the tools needed, including the Wire binary at the specified version.
+Running `make run` calls `make gen-go` on the first run. The `gen-go` in turn calls the wire binary and generates the code in [`wire_gen.go`](/pkg/server/wire_gen.go) and [`wire_gen.go`](/pkg/cmd/grafana-cli/runner/wire_gen.go). The wire binary is installed using [`bingo`](https://github.com/bwplotka/bingo) which downloads and installs all the tools needed, including the Wire binary at the specified version.
```

Comment:
not sure if "Wire" should be capitalized to be consistent with earlier versions.

Reply:
yes, i believe it should, according to Google's documentation (but they're not 100% consistent themselves)